src/init.cpp

```diff
@@ -358,6 +358,11 @@ void SetupServerArgs()
     gArgs.AddArg("-minimumchainwork=<hex>", strprintf("Minimum work assumed to exist on a valid chain in hex (default: %s, testnet: %s)", defaultChainParams->GetConsensus().nMinimumChainWork.GetHex(), testnetChainParams->GetConsensus().nMinimumChainWork.GetHex()), true, OptionsCategory::OPTIONS);
     gArgs.AddArg("-par=<n>", strprintf("Set the number of script verification threads (%u to %d, 0 = auto, <0 = leave that many cores free, default: %d)",
         -GetNumCores(), MAX_SCRIPTCHECK_THREADS, DEFAULT_SCRIPTCHECK_THREADS), false, OptionsCategory::OPTIONS);
+    gArgs.AddArg("-mitigatespectre", strprintf("Attempt to enable per process spectre mitigations (default: %u%s)", DEFAULT_MITIGATE_SPECTRE, (
```

Comment:
Might want to add that this works around a CPU vulnerability, for users that don't know what `spectre` is.

Reply:
Hmm, I wonder if it should be future-proofed by making it applicable to any other vulns we could mitigate later.

Reply:
I think there will soon be multiple spectre related prctl controls such as [this](https://lkml.org/lkml/2018/11/21/179).